x-pack/plugins/security_solution/public/timelines/components/save_status/index.test.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
```

Comment:
Thanks reminded me of something to test and found a bug 😅  https://github.com/elastic/kibana/issues/173968